app/src/main/java/me/ccrama/redditslide/Fragments/CommentPage.java

```diff
@@ -316,7 +314,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         toolbar = v.findViewById(R.id.toolbar);
         toolbar.setPopupTheme(new ColorPreferences(getActivity()).getFontStyle().getBaseId());
 
-        if (!SettingValues.fabComments) {
+        if (!SettingValues.fabComments || archived || np || locked) {
```

Comment:
This is where I am trying to prevent the reply FAB from showing for posts where you cannot reply. I only vaguely know what I did here, but it seems right.

Reply:
This would be correct, as the archived/np/locked are set onCreate, and this is in onCreateView!